Assets/Scripts/JumpPlayer.cs

```diff
@@ -0,0 +1,62 @@
+﻿using System.Collections;
+using System.Collections.Generic;
+using UnityEngine;
+
+public class JumpPlayer : MonoBehaviour
+{
+    [SerializeField]
```

Comment:
Marking this one as an example, you should put [Tooltip("Whatever the heck this thing does")] on a line adjacent to [SerializeField] so the fields have mouseover text in-editor.